src/wiremind_kubernetes/kubernetes_client_additional_arguments.py

```diff
@@ -42,30 +46,30 @@ def fn(*args, **kwargs):
 
 
 class CoreV1ApiWithArguments(ClientWithArguments):
-    def __init__(self, *args, dry_run: bool = False, **kwargs):
+    def __init__(self, *args, dry_run: bool = False, pretty: bool = True, **kwargs):
```

Comment:
you don't pass pretty to `super().__init__()`? could you add/adapt a unit test for this? (case pretty=False)

Could you also create an issue here to clean the `*args, **kwargs`? it'd a breaking change, in case they are "used" somewhere.

Reply:
mb and yes its done